scico/solver.py

```diff
@@ -257,20 +257,20 @@ def cg(
 ) -> Union[JaxArray, dict]:
     r"""Conjugate Gradient solver.
 
-    Solve the linear system :math:`A\mb{x} = \mb{b}` via the conjugate
-    gradient method.
+    Solve the linear system :math:`A\mb{x} = \mb{b}`, where :math:`A` is
+    positive definite, via the conjugate gradient method.
```

Comment:
needs to be symmetric, too

Reply:
Confusingly, I think most authors make positive definite operators a subset of self-adjoint operators, so positive definite -> symmetric.